src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -353,6 +420,23 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 		}
 	}
 
+	public restore(): void {
+		this._instantiationService.invokeFunction(showRestore, this.connectionManagementService.connectionInfo.connectionProfile);
+	}
+
+	public newQuery(): void {
+		this._instantiationService.invokeFunction(openNewQuery);
+	}
+
+	public newNotebook(): Promise<void> {
+		return this.commandService.executeCommand('mssqlCluster.task.newNotebook', this.connectionManagementService.connectionInfo.connectionProfile);
+	}
+
+	public manageExtensions(): void {
+		// TODO: implement what to do here
+		console.error('clicked manage extensions');
```

Comment:
Just have this no-op for now. (and I'd suggest just hiding it until we have it doing something so people aren't confused)